src/sendmail/mail_sender.py

```diff
@@ -1,12 +1,14 @@
 import os
 import mailtrap as mt
 from jinja2 import Environment, FileSystemLoader, select_autoescape
+from src.demolition_spider import BuildingServicesSearchResult
```

Comment:
This is causing a circular import error. Fortunately we only need this for type checking so can do this: https://stackoverflow.com/questions/39740632/python-type-hinting-without-cyclic-imports

Reply:
I think you guys already wrote about removing send_mail import from demolition_spider so maybe that change will fix this first lol

Reply:
Took that solution, that makes the most sense to me, good job :).